README.md

```diff
@@ -43,7 +43,7 @@ and ensuring that `$GOPATH/bin` is added to your `$PATH`.
 
 ## Project status
 
-As of version v0.3.0, Wire is *beta* and is considered feature complete. It
+As of version v0.5.0, Wire is *beta* and is considered feature complete. It
```

Comment:
Hey @realsangil, thanks for this change. Although I think the original sentence better reflects the state of the project since it has entered beta and considered feature complete since 0.3.0

Reply:
Oops, I misunderstood that sentence.
I'll close this PR.
Thank you!